README.md

```diff
@@ -85,6 +85,8 @@ If you're looking contribute to Bullet Train, you should "Fork" this template re
 6. Run the setup script:
 
     ```
+    bundle install
+    corepack enable

```

Comment:
I think we should modify `bin/setup-scripts/check_yarn.rb` to run `corepack enable` instead of adding it to the docs here.